fmriprep/data/boilerplate.bib

```diff
@@ -292,3 +292,14 @@ @article{afni
     volume = 10,
     year = 1997
 }
+
+@article{posse_t2s,
+    author = {Posse, Stefan and Wiese, Stefan and Gembris, Daniel and Mathiak, Klaus and Kessler, Christoph and Grosse-Ruyken, Maria-Liisa and Elghahwagi, Barbara and Richards, Todd and Dager, Stephen R. and Kiselev, Valerij G.},
+    doi = {10.1002/(SICI)1522-2594(199907)42:1<87::AID-MRM13>3.0.CO;2-O},
+    journal = {Magnetic Resonance in Medicine},
+    number = 1,
+    pages = {87-97},
+    title = {Enhancement of BOLD-contrast sensitivity by single-shot multi-echo functional MR imaging},
```

Comment:
Please use braces around acronyms so that capital letters are kept: `{BOLD}` and `{MR}`.